cli/functionary/login.py

```diff
@@ -16,15 +17,7 @@ def login_cmd(ctx, user, password, host):
     for other functionary commands to use to communicate with the server.
     """
     login_url = f"{host}/api/v1/api-token-auth"
-    success, message = login(login_url, user, password)
-
-    # check status code/message on return then exit
-    if success:
-        click.echo("Login successful!")
-    else:
-        click.secho(
-            message,
-            err=True,
-            fg="red",
-        )
-        ctx.exit(1)
+    # try to login, click will raise error from inside login if something goes wrong
+    login(login_url, user, password)
+    click.echo("Login successful!")
+    save_config_value("host", host)
```

Comment:
There's a bit of inconsistency here in that the token value gets saved inside the `login` call, but the host is saved here after the call.  It seems like the desired behavior is the same: If the login is successful, save the config values.

I think what I would probably do is make it so that the login function doesn't have to concern itself with the config.  Instead, make it return the token upon successful login.  Then save both the token and the host here.